mxcube-launcher

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/zsh
```

Comment:
IMO this is a site-specific shell script that should not be included in the code repository.